### PR TITLE
replace api_endpoint with host and port, rework review request

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

File: lib/castle/commands/review.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Castle
+  module Commands
+    class Review
+      def initialize(_context)
+      end
+
+      def build(options = {})
```
Comment (lluft) [inline lib/castle/commands/review.rb]: why add options instead of just passing the `review_id`?

Comment (nijikon) [reply lib/castle/commands/review.rb]: We should be consistent with our interface. It should be the same for every endpoint.

Comment (lluft) [reply lib/castle/commands/review.rb]: well the other ones are all post endpoints and this is a get. I would argue that they are semantically different. `Stripe::Charge.retrieve("ch_18atAXCdGbJFKhCuBAa4532Z")` this is how Stripe does it. I think it is more self-explanatory

Comment (nijikon) [reply lib/castle/commands/review.rb]: Fair enough.

File: lib/castle/commands/review.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Castle
+  module Commands
+    class Review
+      def initialize(_context)
+      end
```
Comment (lluft) [inline lib/castle/commands/review.rb]: why not just remove the `_context` in init if it's not used?

Comment (nijikon) [reply lib/castle/commands/review.rb]: All `Commands` have the same interface, so even if it is not used, it should be there.

**Pull request merged**